code/modules/maps/helper_landmarks.dm

```diff
@@ -78,8 +78,8 @@ INITIALIZE_IMMEDIATE(/obj/abstract/landmark/map_load_mark)
 	var/turf/wall/simulated_wall = get_turf(src)
 	if(istype(simulated_wall))
 		simulated_wall.dismantle_wall(TRUE, TRUE, TRUE)
-	else if(istype(simulated_wall, /turf/exterior/wall))
-		var/turf/exterior/wall/exterior_wall = simulated_wall
+	else if(istype(simulated_wall, /turf/wall/natural))
+		var/turf/wall/natural/exterior_wall = simulated_wall
 		exterior_wall.dismantle_wall(TRUE)
```

Comment:
This might be able to be simplified somehow now, but I'm not totally sure. Approving since, you know, I have no clue.

Reply:
Important to note the args, though. That's always been the main difference.